lib/manager/poetry/extract.spec.ts

```diff
@@ -56,7 +56,11 @@ describe('lib/manager/poetry/extract', () => {
     it('extracts multiple dependencies', () => {
       const res = extractPackageFile(pyproject1toml, filename);
       expect(res.deps).toMatchSnapshot();
-      expect(res.deps).toHaveLength(9);
+      expect(res.deps).toHaveLength(10);
+      expect(res.compatibility).toEqual({
+        poetry: 'poetry>=1.0 wheel',
```

Comment:
Does this format work when put into `pip install` ?

Reply:
sure, tested locally:

![image](https://user-images.githubusercontent.com/1798109/80947509-93d37e00-8df0-11ea-94d2-b4112d46a70b.png)